src/pages/storey/containers/item.mdx

```diff
@@ -5,3 +5,87 @@ tags: ["storey", "containers"]
 import { Callout } from "nextra/components";
 
 # Item
+
+An [`Item`] is the simplest container. It stores a single value at a specific key.
+
+An `Item` that's never been `set` is empty, and will return `None` upon a `get`.
+
+<Callout>
+  For a more in-depth look at the `Item` container, check out the [API documentation].
+</Callout>
+
+## Usage examples
+
+### Saving an admin address
+
+```rust template="storage"
+use cw_storey::containers::Item;
+use cw_storey::CwStorage;
+
+let admin: Item<String> = Item::new(0);
```

Comment:
This API looks unintuitive as it appeary as if you use value 0.

What about making that key a byte string an call it like this?

```suggestion
let admin: Item<String> = Item::new(b"\0");
```

Reply:
I wanted to encourage using a single byte here. Also, allowing users to use dynamic strings create more potential for collisions. This stuff isn't length-prefixed at all.

Reply:
As an alternative, we could rename the constructor maybe?

Reply:
I'd actually leave an API as it is, but encourage different usage of it. Magic numbers seems not ok, but what seems ok to me is:

```
const MY_FANCY_ITEM: u64 = 0;
let admin = Item<String> = Item::new(MY_FANCY_ITEM);
```

I am also wondering if some proc-macro magic could incrementally generate indexes so we could not even pass it, and guarantee the same index through compilation, but I would be super careful of that - we want to avoid situations where reordering entities in code changes indices.

Reply:
The const thing sounds good.

Long-term, I was thinking of a proc macro. I didn't work on it in the first iterations since it would have been a bit overwhelming for me to tackle that and the lib design at the same time.

My idea for the proc macro would **still** have explicit indexes, e.g.:

```rs
#[derive(Storable)]
struct MyData {
    #[ix(0)]
    foo: Item<u32>, // maybe the `Item` wouldn't even be needed here
    #[ix(1)]
    bar: Map<String, ...>
}
```

This is inspired by how RPC systems tend to work (AFAIK). For example, [Thrift forces you to attach IDs to struct fields](https://raw.githubusercontent.com/apache/thrift/master/test/ThriftTest.thrift). It's "inconvenient", but it should solve a lot of potential problems with reordering, adding, and removing fields and would force contract devs to think changes through that would otherwise bite them in the butt very badly.

Reply:
Reference: https://github.com/CosmWasm/storey/issues/8